google/cloud/forseti/services/scanner/dao.py

```diff
@@ -47,7 +47,7 @@ class Violation(base):
         __tablename__ = violations_tablename
 
         id = Column(Integer, primary_key=True)
-        model_handle = Column(String(256))
+        inventory_id = Column(String(256))
```

Comment:
Should this be inventory_id or inventory_index_id?  I think it's better to have this as a foreign key and reference back to the inventory/inventory_index table.

Reply:
This is a good point.  I originally had it as `inventory_index_id`, but then thought to be consistent with the rest of the usage of `inventory_id` elsewhere in the code base.  Do you think that we should change this everywhere?  As there is a difference between the two.

Reply:
After more discussions, this should be better as ```inventory_index_id```, so that we do not confuse it with ```inventory_id```